coalib/output/ConsoleInteraction.py

```diff
@@ -338,6 +340,13 @@ def print_results_formatted(log_printer,
 
             for range in result.affected_code:
                 format_args['affected_code'] = range
+
+                if file_dict:
```

Comment:
that test assumes that this function doesn't need a file dict which is no longer true so the test should be corrected